Project.toml

```diff
@@ -3,10 +3,11 @@ uuid = "c7f686f2-ff18-58e9-bc7b-31028e88f75d"
 keywords = ["markov chain monte carlo", "probablistic programming"]
 license = "MIT"
 desc = "Chain types and utility functions for MCMC simulations."
-version = "5.1.0"
+version = "6.0.0"
 
 [deps]
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
+ArraysOfArrays = "65a8f2f4-9b39-5baf-92e2-a9cc46fdf018"
```

Comment:
Why is this needed? Weights should not be related to ArraysOfArrays (which BTW is only one possible approach to arrays of arrays).

Reply:
Accidental, thanks.